### PR TITLE
Fix eternal reconnection issue

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -28,7 +28,6 @@ addCommandAlias(
 ThisBuild / description                         := "Explore"
 Global / onChangedBuildSource                   := ReloadOnSourceChanges
 ThisBuild / scalafixDependencies += "edu.gemini" % "lucuma-schemas_3" % Versions.lucumaSchemas
-ThisBuild / scalafixScalaBinaryVersion          := "2.13"
```
Comment (cquiroz) [inline build.sbt:31]: Why was this removed?

**Review by cquiroz — APPROVED** [review]
LGTM

File: build.sbt (hunk 1 above)
Comment (rpiaggio) [reply build.sbt:31]: It was causing a deprecated warning with the new scalafix.
